submissions/LGeorge007/memory-pair-game/script.js

```diff
@@ -0,0 +1,93 @@
+let initialArray = ["img/cartman.jpg",
+    "img/chief.jpg",
+    "img/sten.jpg",
+    "img/kyle.jpg",
+    "img/kenny.jpg"
+];
+let arrOfUnits;
+const imgCover = `img/cover.jpg`;
+const gameField = document.querySelector("#gameField");
+let isLocked = false;
+let firstCard;
+let secondCard;
+let isSecondCard = false;
+let unopenedCards;
+
+//feed him array & any even number
+const duplicateElements = function(array, times) {
+    return array.reduce((res, current) => {
+        return res.concat(Array(times).fill(current));
+    }, []);
+};
+
+const shuffle = function(arr) {
+    const result = [];
+    while (arr.length > 0) {
+        let random = Math.floor(Math.random() * arr.length);
+        let elem = arr.splice(random, 1)[0];
+        result.push(elem);
+    }
+    return result;
+};
+
+const checkCard = function({ target }) {
```

Comment:
[separation of concerns](https://en.wikipedia.org/wiki/Separation_of_concerns)

too big function. what it do ? check cards, check win condition etc.
 So it better to have smaller, more abstract functions, than complex like this one